src/test/cpp/util/transformer.cpp

```diff
@@ -116,14 +116,25 @@ void Transformer::createSedCommandFile(const std::string& regexName,
 
 	std::string tmp;
 
+	auto sedsaniziter = [] (const std::string in, const std::string sedseperator = "Q") {
```

Comment:
Shouldn't the arguments better be `const std::string&`? Especially as you are copying `in` again.

Reply:
you're right, fixing ...